CHARTER.adoc

```diff
@@ -0,0 +1,195 @@
+
+
+
+== Business Value Proposition
+
+In the geospatial world, new cloud native data formats are emerging. Zarr is a generic data format for n-dimensional arrays that enables access to data in compressed chunks of the original array and has become increasingly popular to use for geospatial purposes. In June 2022, the OGC endorsed a community standard of Zarr V2.0 (https://zarr.readthedocs.io/en/stable/spec/v2.html). The purpose of this charter is to adopt a more explicit GeoZarr as an OGC Standard which would provide guidance to standardize the approach for encoding various aspects of geospatial data in zarrs.
+
+== Scope of Work
+
+The goal of the GeoZarr specification is to establish flexible and inclusive conventions for the Zarr cloud-native format, specifically designed to meet the diverse requirements within the geospatial domain. These conventions aim to provide a clear and standardized framework for organizing and describing data, ensuring unambiguous representation. 
+
+The objectives of GeoZarr conventions includes:
+
+1. Compatibility: Ensuring easy compatibility with popular mapping and data analysis tools such as GDAL, Xarray, ArcGIS, and QGIS, enabling seamless integration into existing workflows.
+2. Visualisation: Simplifying the creation and display of geospatial data in web browsers without the need for complex workarounds, making geospatial information more accessible to users.
+3. Dimensions: Supporting multidimensional data, such as hyperspectral and altitude information, to address diverse geospatial data requirements.
+4. Data Discovery: Providing metadata for discovering, accessing, and retrieving the data, including subsets or different arrangements of the data.
+5. Mixing Data: Facilitating the combination of different types of geospatial data, including satellite images, elevation maps, and weather models, to create comprehensive and informative datasets.
+8. Flexibilty: Allowing scientists and researchers to work with diverse data types and projections in their preferred software and programming languages, promoting flexibility and adaptability in geospatial data processing and analysis.
+
+Specifically, the convention should provide guidance to standardize the approach for encoding various aspects of geospatial data, including:
+
+* Multiple related variables with heterogeneous coordinates (e.g., children or linked datasets)
+* Multiple resolutions of the data, possibly leveraging multiscale drafts
+* Data subsets that are only available at certain resolutions
+* Multiple projections
```

Comment:
```suggestion
```

Removing this because I don't understand what is meant here. Do we want to store multiple projections of the same data? That feels like unnecessary scope creep.

Reply:
That's maybe not a major/primordial aspect but as some big data cubes will be migrated to cloud-native data (for acting like data access and map tile services), it will typically store in the zarr store data in a world-wide projection, and in a local projection (use to fasten processing / visualisation of the data).

Indeed, it is important to identify secondary aspects that may not be immediately addressed in the initial version.

Reply:
Removing per Ryan's suggestion